docs/how-to/ChIP_seq_genomic_ranges_tutorial.ipynb

```diff
@@ -0,0 +1,2733 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ensdb.list_columns()  # you can custom select the columns output if exon, protein, or sequence information that is not included by default
</pre><p>I can't seem to paste an image here, but this comment is difficult to read when rendered (see <a href="https://genomic-features--65.org.readthedocs.build/en/65/how-to/ChIP_seq_genomic_ranges_tutorial.html#download-ensembl-annotations-using-genomic-features" target="_blank">here</a>). Can it be broken up or written in markdown text?</p><p> </p><p>I think the output of this line should also probably be show if this line is included.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/genomic-features/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>